docs/management/release.rst

```diff
@@ -10,15 +10,16 @@ person doing the release must ensure that the version of the new catalog matches
 following all steps below. Steps 1 and 2 below should be done in a PR and merged before commencing step 3. If the release does 
 not include an update to the catalog on Gadi, skip the first two steps below:
 
-#. [IF UPDATING THE CATALOG] Create a new version of the catalog on Gadi (this will take about 1 hour)
-   ::
-      $ export RELEASE=vX.X.X
-      $ cd bin
-      $ qsub -v version=${RELEASE} build_all.sh
+#. [IF UPDATING THE CATALOG] Create a new version of the catalog on Gadi (this will take about 1 hour)::
+
+     $ export RELEASE=vX.X.X
+     $ cd bin
+     $ qsub -v version=${RELEASE} build_all.sh
+
    .. note:: 
-      If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the :code:`access_nri_intake.source` and :code:`access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of :code:`access_nri_intake`
-      ::
-         $ python3 -c "from access_nri_intake import source, catalog"
+      If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the :code:`access_nri_intake.source` and :code:`access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of :code:`access_nri_intake`::
+
```

Comment:
I'll assume you meant to do this and you don't want the code blocks.

Reply:
The code block should still be there (I moved the `::` to the previous line). Things were rendering correctly, but Sphinx was complaining about indentation and this seemed to fix that.

Reply:
Soz. Thought I'd check for the code block thingoes at the end of the line. I even have my glasses on, so no excuses on that front.